src/main/java/com/monumental/controllers/MonumentController.java

```diff
@@ -21,4 +23,15 @@ public Monument createMonument(Monument monument) {
     public Monument getMonument(@PathVariable("id") Integer id) {
         return this.monumentService.get(id);
     }
+
+    @GetMapping("/api/monuments")
+    public List<Monument> getAllMonuments() {
+        return this.monumentService.getAll();
+    }
+
+    @PutMapping("/api/monument/{id}")
+    public Monument updateMonument(@PathVariable("id") Integer id, Monument monument) {
```

Comment:
This method completely replaces the row in the database with whatever JSON you provide. If you don't include a field it will be set to null even if some data exists for it in the database. I think in all cases React will have all the data so this isn't an issue, but keep in mind while testing